### DISCUSSION TITLE
Error installing AdminLTE 3.1.0

### DISCUSSION THREAD

**Discussion by CTInnovacion** [ColorlibHQ/AdminLTE#3616] [General]
Hi.

I have tried to install AdminLTE 3.1.0 on Joomla! 3.9.26 Stable [ Amani ] 13-April-2021 15:00 GMT. I was not possible, because the system info send the follow error:
Advertencia
JInstaller: :Instalar: No se ha podido encontrar el archivo XML de configuración.
Error
No se ha podido detectar el archivo de manifiesto

Although he tries, there is no copy of files in the folders of the system, it is as if nothing had happened, taking about 15 minutes to throw me the error.

Please, I would appreciate your help with this problem.

Thanks in advance.
![error AdminLTE 3](https://user-images.githubusercontent.com/81710706/115538529-99c0aa00-a261-11eb-94ab-20a3d79bad4e.png)

**Comment by danny007in** [comment]
in English
many of don't know, many languages

**Reply by CTInnovacion** [reply]
What in English? The system give me this, like you can see. Remember that google translator exist.

Advertencia
JInstaller: :Instalar: No se ha podido encontrar el archivo XML de configuración.
Error
No se ha podido detectar el archivo de manifiesto

Translation:

Warning
JInstaller:: Install: The configuration XML file could not be found.
Error
The manifest file could not be detected

**Reply by danny007in** [reply]
OK this translation Good 👍🏼 , but not every language is supported like this.

**Reply by REJack** [reply]
Google Translate isn't a good translator if there are foreign words based to a technology or something else, that's why we won't give support if you not ask in english.

We don't deliver a manifest file that's a Java error I guess.

**Reply by CTInnovacion** [reply]
I have installed java and the result is the same.

I have tried to install other dashboard, and is the same result.

I think that is a joomla problem.

**Reply by CTInnovacion** [reply]
Well ...

I have done, whatever that you can found on internet, to solve it. But, is not possible.

I think, that is a Jommla problem, but AdminLTE is yours. For this reason, you must try to help me, please.

Thanks.
